js/renovation/ui/form/form.tsx

```diff
@@ -7,15 +7,39 @@ import { FormProps } from './form_props';
 
 import { combineClasses } from '../../utils/combine_classes';
 import { Widget } from '../common/widget';
+import { Scrollable } from '../scroll_view/scrollable';
 
 export const viewFunction = (viewModel: Form): JSX.Element => {
-  const { aria, cssClasses, restAttributes } = viewModel;
-  return (
-    <Widget
-      aria={aria}
-      classes={cssClasses}
-      {...restAttributes} // eslint-disable-line react/jsx-props-no-spreading
-    />
+  const {
+    aria,
+    cssClasses,
+    props: {
+      scrollingEnabled,
+      useNativeScrolling,
+    },
+    restAttributes,
+  } = viewModel;
+  return (scrollingEnabled
+    ? (
+      <Scrollable
+        aria={aria}
+        classes={cssClasses}
+        useNative={!!useNativeScrolling}
```

Comment:
Without useSimulatedScrollbar={!useNativeScrolling} on android devices in case when useNative=true scrollbars will appear. I think it is not expected for customer.